src/main/java/emu/grasscutter/game/avatar/Avatar.java

```diff
@@ -753,7 +753,8 @@ public boolean upgradeSkill(int skillId) {
     }
 
     public boolean setSkillLevel(int skillId, int level) {
-        if (level < 0 || level > 15) return false;
+        if (level < 1) level = 1;
+        if (level >15) level = 15;
```

Comment:
Shouldn't cap levels here, also setting to 0 is a feature that shouldn't be removed.

Reply:
Just revert the two line change :)